code/game/machinery/ds13/bpl/recycling_tank.dm

```diff
@@ -13,7 +13,7 @@
 	var/list/content_atoms = list()
 
 
-	var/breakdown_rate = 0.001	//Remove this many units of biomass per tick, and convert it into purified biomass
```

Comment:
I said double, you've increased it by 3000%

Reply:
I tried 0.003 and it gives 10 biomass per minute which means that we can make 1 limb in 10 minutes